app/src/main/java/ch/epfl/sdp/artificial_intelligence/UnboundedArea.java

```diff
@@ -0,0 +1,8 @@
+package ch.epfl.sdp.artificial_intelligence;
+
+public class UnboundedArea implements Boundable {
```

Comment:
Why do we need the unboundedArea?

Reply:
Can be useful as default value, instead of Null. And also for test purposes.